### ISSUE TITLE
"curt" error logs (Full Node)

### ISSUE THREAD

**Issue by dkaraush** [ton-blockchain/ton#200]
I'm trying now to start Full Node and sync it with testnet2.

I have updated ton code (`git pull`) and noticed new commit: 090e0c1.
After trying to run validator-engine, it shows me an error:
`[ 1][t39][1574974188.345818043][validator-engine.cpp:930][!validator-engine]	curt: 1573821854 1573921854`

As I see, this error log is new, so it wasn't ever mentioned anywhere.
Could you, please, tell me what I am doing wrong.

The way I run full node:
(`/tondb/` is just a link to `/mount/sdisk2/tondb/`, `sdisk2` - 832GB SSD, `sdisk3` - 8TB HDD)
```
rm -rf /tondb/*
rm -rf /mount/sdisk3/tondb-archive/*
ln -s /mount/sdisk3/tondb-archive/ /tondb/archive
validator-engine -C ton-global.config.json --db /mount/sdisk2/tondb/ --ip {IP}:8763 -S 3600000 -v 3 -t 64
cp ~/keyring/5F55xxxEF3 /mount/sdisk2/tondb/keyring/5F55xxxEF3
cp ~/keyring/26xxx9 /mount/sdisk2/tondb/keyring/26xxx9
nano /mount/sdisk2/tondb/config.json # adding liteclient and control elements
validator-engine -C ton-global.config.json --db /mount/sdisk2/tondb/ --ip {IP}:8763 -S 3600000 -v 3 -t 64
```

Logs:
```
... no nodes ...

[ 2][t 8][1574974167.949515343][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,0)]	failed to download state : [Error : 651 : no nodes]
[ 2][t46][1574974168.468820095][full-node-shard.cpp:382][!manager]	getnextkey: [Error : 651 : not inited]
[ 2][t 9][1574974168.951301575][download-state.cpp:232][!downloadstatereq(-1,8000000000000000,0)]	failed to download state : [Error : 651 : no nodes]
[ 2][t60][1574974171.733660936][full-node-shard.cpp:382][!manager]	getnextkey: [Error : 651 : not inited]
[ 2][t47][1574974173.881163120][full-node-shard.cpp:382][!manager]	getnextkey: [Error : 651 : not inited]
[ 2][t57][1574974173.958121300][full-node-shard.cpp:382][!manager]	getnextkey: [Error : 651 : not inited]
[ 2][t10][1574974174.114409924][manager-init.cpp:183][!reiniter]	last key block is [ w=-1 s=9223372036854775808 seq=156120 E/KNbP3g9n/cbyBRwFjhHYEpkAGBYteT8eVMXWN8ywc= ix2vlUkqXF1X2XdNbaBlnzq3a5kMG6G+WUkqLOoxRlI= ]
[ 2][t57][1574974178.021271229][manager-init.cpp:183][!reiniter]	last key block is [ w=-1 s=9223372036854775808 seq=232888 XpgvLWausnPhCMy8gAERLUzghxHfRkXx7HZPGoriRaU= cJludIwUQ1PZNjURn5kH2/Ba7/dGlTWmcr2AsDNBzmk= ]
[ 2][t41][1574974179.400242090][full-node-shard.cpp:382][!manager]	getnextkey: [Error : 651 : not inited]
[ 2][t 2][1574974180.037684441][full-node-shard.cpp:382][!manager]	getnextkey: [Error : 651 : not inited]
[ 2][t40][1574974182.027387857][manager-init.cpp:183][!reiniter]	last key block is [ w=-1 s=9223372036854775808 seq=309902 nS+PcNQBu8zyiof3N4dX06XuVptHtTE426yeL06Ox5I= JgMRtQjcHmRV9f+QVtsWAPVC7hqpNsvMALwgCiSIr8k= ]
[ 2][t16][1574974183.726913214][manager-init.cpp:183][!reiniter]	last key block is [ w=-1 s=9223372036854775808 seq=350651 qyy24m9EBMWl6y+bSCWrLKBWBPp1JYKYj5ERiMDoJRY= xUOMzYoQVEXkqMEot+WUiCAtZp+iJqY7kgEm0tm0jVo= ]
[ 2][t 8][1574974184.292372465][full-node-shard.cpp:382][!manager]	getnextkey: [Error : 651 : not inited]
[ 3][t15][1574974184.851890087][manager-init.cpp:220][!reiniter]	key block candidate: seqno=350651 is_persistent=true ttl=1574967929 syncbefore=3600000.000000
[ 3][t15][1574974184.851958990][manager-init.cpp:225][!reiniter]	ignoring: too new block

... key block candidates ...
 
[ 2][t15][1574974184.853700161][manager-init.cpp:244][!reiniter]	best handle is [ w=-1 s=9223372036854775808 seq=0 F6OpKZKqvqeFp6CQmFomXNMfMj2EnaUSOXN+Mh+wVWk= XplPz01CXAps5qeSWUtxcyBfdAo5zVb1N979KLSKD24= ]
[ 2][t12][1574974185.333090305][manager.cpp:1574][!manager]	applied masterchain block [ w=-1 s=9223372036854775808 seq=0 F6OpKZKqvqeFp6CQmFomXNMfMj2EnaUSOXN+Mh+wVWk= XplPz01CXAps5qeSWUtxcyBfdAo5zVb1N979KLSKD24= ]
[ 3][t12][1574974185.333184719][manager.cpp:1459][!manager]	groups=0 seqno=0 our_seqno=0
[ 3][t29][1574974185.334407806][manager.cpp:1459][!manager]	groups=0 seqno=0 our_seqno=0
[ 3][t56][1574974185.441139221][manager.cpp:1492][!manager]	downloaded archive slice: /mount/sdisk3/tondb-tmp/tmpXXXXU3LjNR
[ 3][t26][1574974185.441243172][import-db-slice.cpp:359][!archiveimport]	[Error : 651 : archive does not contain any masterchain blocks]
[ 3][t20][1574974185.441308022][manager.cpp:1510][!manager]	checked downloaded archive slice: mc_top_seqno=0 shard_top_seqno_=0
[ 3][t18][1574974185.441880226][manager.cpp:1459][!manager]	groups=0 seqno=0 our_seqno=0
[ 1][t39][1574974188.345818043][validator-engine.cpp:930][!validator-engine]	curt: 1573821854 1573921854
[ 1][t41][1574974189.976258993][validator-engine.cpp:930][!validator-engine]	curt: 1573821854 1573921854
[ 1][t20][1574974191.147016287][validator-engine.cpp:930][!validator-engine]	curt: 1573821854 1573921854
[ 1][t45][1574974192.297338486][validator-engine.cpp:930][!validator-engine]	curt: 1573821854 1573921854
```

Server info:
`uname --all`: `Linux ton 4.19.0-6-amd64 #1 SMP Debian 4.19.67-2+deb10u2 (2019-11-11) x86_64 GNU/Linux`
cmake version 3.13.4
OpenSSL 1.1.1d  10 Sep 2019

2 processors (Intel Xeon E5-2630 v4) with 8 cores each; 32GB RAM, 832GB SSD, 8TB HDD.
Real traffic usage while working validator about 10-20Mbit/s in and out.

**Comment by phil-dolgolev** [comment]
I'm have the similar problem! Looks like need to try rebuild binaries with previous commit :(

**Comment by phil-dolgolev** [comment]
@dkaraush can you find in logs ?
`[ 3][t 1][1575285179.218214273][import-db-slice.cpp:359][!archiveimport]	[Error : 0 : failed to parse block id]`

**Comment by Kyborg2011** [comment]
@phil-dolgolev @dkaraush Hi, guys! I have this problem, too.
@ton-blockchain I think it's a bug. I have tried anything, but still a validator-engine firstly downloading ~400-800 mb of data, then throw an error "failed to parse block id", then a db/tmp dir is cleansed and again a full node tried to download ~400-800 mb of data, throw an error, cleansed and so on, by a infinite cycle...

**Comment by ton-blockchain** [comment]
It indeed was a bug. Should be fixed in new commit ceaed40ac4871ffaeb5b493586ab6f0899fd490d

**Comment by Kyborg2011** [comment]
Yes, it's working right now ) I have tried - it seems everything works!
